packages/api/repository/PortfolioRepository.ts

```diff
@@ -0,0 +1,55 @@
+import { EntityRepository, Repository, InsertResult, UpdateResult } from "typeorm";
+import { IPortfolio, IPortfolioCreate, Portfolio } from "../../orm/entity/Portfolio";
+
+@EntityRepository(Portfolio)
+export class PortfolioRepository extends Repository<Portfolio> {
+  // GET portfolios/:id
+  getPortfolio(id: string): Promise<Portfolio> {
+    return this.findOneOrFail({
+      select: [
+        "id",
+        "name",
+        "description",
+        "owner",
+        "csp",
+        "dodComponents",
+        "portfolioManagers",
+        "createdAt",
+        "updatedAt",
+        "archivedAt",
+        "administrators",
+      ],
+      where: { id },
+    });
+  }
+
+  // GET all portfolios
+  getPortfolios(): Promise<[Array<Portfolio>, number]> {
+    return this.createQueryBuilder("portfolio")
+      .select(["portfolio.name", "portfolio.id", "portfolio.createdAt", "portfolio.updatedAt", "portfolio.archivedAt"])
+      .getManyAndCount();
+  }
```

Comment:
I ended up changing this to `.getMany()` because `.getManyAndCount()` seemed to be unnecessary and dealing with the return structure seemed bulky (see changes in `EnvironmentRepositroy` on commit https://github.com/dod-ccpo/atat-web-api/pull/537/commits/0afbda327ab96b1a102cad2ec3f6dcbb076ac8f9). If an array is returned we can simply use the `.length` property to get the count. 

Both can be used, but I thought this could open a small discussion conversation as if we want `.getMany()` or `.getManyAndCount()`.

Reply:
That change and explanation makes sense.  Changed in 34db71956811264f3ed5daa2c1e79dccf7d2bebf